tests/testthat/test-run.R

```diff
@@ -138,6 +138,20 @@ test_that("can run orderly with parameters, without orderly", {
 })
 
 
+test_that("can ruin orderly with parameters, without orderly, globally", {
```

Comment:
```suggestion
test_that("can run orderly with parameters, without orderly, globally", {
```

"can ruin orderly" :laughing:

Reply:
Actually what does "without orderly" mean here?

Reply:
means without orderly_run (i.e., interactively)